src/components/TablePerson/TablePerson.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+export const TablePerson: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const { name, sex, born, died, motherName, fatherName, mother, father } =
+    person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames('', {
+        'has-background-warning': person.slug === slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
```

Comment:
You have a constant for route `people` so use it

Reply:
Done